write_asv_machine.py

```diff
@@ -0,0 +1,33 @@
+import copy
+import hashlib
+import json
+import os
+
+from asv.machine import Machine, MachineCollection
+
+
+def write_machine_info():
+    # Hack making sure that similar machines will have same name, so that
+    # we can reuse results from previous runs.
+    if os.path.exists(MachineCollection.get_machine_file_path()):
+        os.remove(MachineCollection.get_machine_file_path())
+
+    machine_info = Machine.get_defaults()
+    # approx ram size to MB
+    if machine_info["ram"]:
+        machine_info["ram"] = int(machine_info["ram"]) // int(1e3)
```

Comment:
This is wildly incompatible. At least you should not change the units. If it is in bytes - keep it in bytes but rounded.